### PR TITLE
[WIP] Fix building with crossbuilder

### PR DESCRIPTION
**Pull request by jonnius**

### CODE REVIEW

**Comment by jonnius** [comment]
I tried to disable the tests as @mardy proposed for the messaging app. But I get:
```
dh build --with translations \
        -DENABLE_TESTS=OFF
   dh_testdir -O-DENABLE_TESTS=OFF
   dh_update_autotools_config -O-DENABLE_TESTS=OFF
   dh_auto_configure -O-DENABLE_TESTS=OFF
dh_auto_configure: invalid or non-existing path to the source directory: ENABLE_TESTS=OFF
debian/rules:8: recipe for target 'build' failed
make: *** [build] Error 2
dpkg-buildpackage: error: debian/rules build gave error exit status 2
```
Do I need to split those dh_* calls as it is done in the messaging app?

File: debian/control [hunk 1]
```diff
@@ -10,13 +10,13 @@ Build-Depends: cmake,
...
                qt5-default,
                qtbase5-dev,
-               libqtdbusmock1-dev,
+               libqtdbusmock1-dev <!nocheck>,
                libqtdbustest1-dev,
-               cppcheck,
+               cppcheck <!nocheck>,
                libgtest-dev,
                google-mock (>= 1.6.0+svn437),
                gsettings-ubuntu-schemas (>= 0.0.2+14.10.20140813),
```
Comment (mardy) [inline debian/control:22]: Please add `<!nocheck>` also to `libqtdbustest1-dev`, `libgtest-dev` and `google-mock`

File: debian/rules [hunk 2]
```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/make -f
 
+ifneq (,$(findstring nocheck,$(DEB_BUILD_OPTIONS)))
+	CONFIGURE_OPTS += -DENABLE_TESTS=OFF
+endif
+
 %:
-	dh $@ --with translations
+	dh $@ --with translations \
+	        $(CONFIGURE_OPTS)
```
Comment (mardy) [inline debian/rules]: By doing so, you are passing the `$(CONFIGURE_OPTS)` to every step of the build, while you need to pass them only to the `auto_configure` one. See the second answer from https://superuser.com/questions/250437/how-to-pass-custom-options-to-configure-when-building-a-package-with-debuild :-)

Comment (mardy) [reply debian/rules]: Or you can try with `DEB_CONFIGURE_EXTRA_FLAGS` as explained [here](https://wiki.debian.org/Multiarch/Implementation).

**Review by mardy — CHANGES_REQUESTED** [review]

File: debian/rules (hunk 2 above)
Comment (mardy) [reply debian/rules]: Forget about the `DEB_CONFIGURE_EXTRA_FLAGS` thing, it looks like it doesn't actually exist. Feel free to continue using `CONFIGURE_OPTS` :-)

File: debian/control (hunk 1 above)
Comment (jonnius) [reply debian/control:22]: Done

File: debian/rules (hunk 2 above)
Comment (jonnius) [reply debian/rules]: Done

**Comment by jonnius** [comment]
Thanks again for your help! It builds now as expected.

**Review by mardy — APPROVED** [review]
Excellent, thanks!

**Pull request merged**